.dprintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://dprint.dev/schemas/v0.json",
+  "projectType": "openSource",
+  "incremental": true,
+  "typescript": {
+  },
+  "json": {},
+  "markdown": {
+  },
+  "rustfmt": {
+    "condense_wildcard_suffixes": true,
+    "format_macro_matchers": true,
+    "merge_imports": true,
+    "use_field_init_shorthand": true,
+    "format_code_in_doc_comments": true,
+    "normalize_comments": true,
+    "wrap_comments": true,
+    "overflow_delimited_expr": true
+  },
+  "includes": ["**/*.{ts,tsx,js,jsx,json,md,rs,toml}"],
+  "excludes": [
+    "**/node_modules",
+    "**/*-lock.json",
+    "**/target",
+    "LICENSE.md",
+    "tests/root.schema.json",
+    "tests/siren.schema.json",
+    "nectar/src/database.rs"
```

Comment:
This is unfortunate for now, here is the explanation: https://github.com/dprint/dprint/issues/320